sparrow.yaml

```diff
@@ -0,0 +1,29 @@
+app-id: com.craigrow.sparrow
+runtime: org.gnome.Platform
```

Comment:
Does this really need GNOME runtime? Are you using in GNOME libraries?

Reply:
thank you! I changed this field - **org.freedesktop.Platform**